libs/api/domains/health-insurance/src/lib/soap/soapClient.ts

```diff
@@ -15,25 +15,39 @@ export class SoapClient {
     baseUrl: string,
     username: string,
     password: string,
+    clientID: string,
+    serviceID: string,
```

Comment:
This code should not be duplicated. The health insurance domain should use `libs/health-insurance`.

Reply:
Done merged